elliottlib/util.py

```diff
@@ -544,21 +544,26 @@ def isolate_timestamp_in_release(release: str) -> Optional[str]:
     return None
 
 
-def get_golang_container_nvrs(nvrs, logger):
+def get_golang_container_nvrs(nvrs: List[Tuple[str, str, str]], logger) -> Dict[str, Dict[str, str]]:
+    """
+    :param nvrs: a list of tuples containing (name, version, release) in order
+    :param logger: logger
+
+    :return: a dict mapping nvr 'name' to a dict containing nvr and go-builder as values
```

Comment:
:return: a dict mapping go_version to a list contains nvrs

Reply:
Fixed!